### DISCUSSION TITLE
direct use of nerdctl with wsl2 backend?

### DISCUSSION THREAD

**Discussion by sixwaaaay** [containerd/nerdctl#1474] [Q&A] [answered]
Is there a way for me to use nerdctl without a redundant use of linux shell usually? I want to use it on powershell of windows 11.

**Comment by yuchanns** [comment]
According to `wsl --help`, there is an argument `--execute` or `-e` that can directly run Linux commands without entering the Linux shell. So I guess `wsl -e nerdctl` is what you want.
One more step, you can use an alias to simplify further:
```powershell
Function NERDCTL {wsl -e nerdctl}
New-Alias -Name nerdctl -Value NERDCTL
```

**Comment by AkihiroSuda** [comment, marked as answer]
Rancher Desktop contains `nerdctl.exe` that wraps nerdctl in WSL2
https://rancherdesktop.io/
